boards/arm/nucleo_l552ze_q/Kconfig.defconfig

```diff
@@ -14,6 +14,9 @@ if TRUSTED_EXECUTION_NONSECURE
 config USE_DT_CODE_PARTITION
 	default y
 
+config TFM_ISOLATION_LEVEL
+	default 2 if BUILD_WITH_TFM
```

Comment:
I think this options depends on BUILD_WITH_TFM so the dependency should be more visible as "depends on".
Otherwise it gives the impression that the option could be defined to a different LEVEL if BUILD_WITH_TFM is not defined....